open-machine-learning-jupyter-book/_toc.yml

```diff
@@ -50,20 +50,24 @@ parts:
 - caption: MACHINE LEARNING BASICS
   numbered: True
   chapters:
-  - file: ml-fundamentals/ml-overview
+  - file: ml-fundamentals/ml-overview.ipynb
+  - file: ml-fundamentals/ml-summary.ipynb
   - file: ml-fundamentals/regression/regression-models-for-machine-learning
     sections:
     - file: ml-fundamentals/regression/tools-of-the-trade
     - file: ml-fundamentals/regression/managing-data
     - file: ml-fundamentals/regression/linear-and-polynomial-regression
     - file: ml-fundamentals/regression/logistic-regression
-  - file: ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model
   - file: ml-fundamentals/classification/getting-started-with-classification
     sections:
```

Comment:
Line 60（- file: ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model） should be retained

Reply:
DONE